src/Common/src/Interop/Unix/libcurl/Interop.libcurl_types.cs

```diff
@@ -29,5 +54,30 @@ internal static partial class CURLcode
         {
             internal const int CURLE_OK = 0;
         }
+
+        // Class for constants defined for the enum curlioerr in curl.h
+        internal static partial class curlioerr
+        {
+            internal const int CURLIOE_OK = 0;
+            internal const int CURLIOE_UNKNOWNCMD = 1;
+            internal const int CURLIOE_FAILRESTART = 2;
+        }
+
+        // Class for constants defined for the enum curliocmd in curl.h
+        internal static partial class curliocmd
+        {
+            internal const int CURLIOCMD_RESTARTREAD = 1;
+        }
+
+        public delegate ulong curl_readwrite_callback(
+            IntPtr buffer,
+            ulong size,
+            ulong nitems,
```

Comment:
These fields are actually size_t, right?  We should follow the convention employed by other interop files, which is to use size_t here, and then at the top of the file use a ```using size_t = System.IntPtr;``` or whatever the correct definition is.  (This will likely all be overhauled, though, as part of @nguerrera's work.)